src/Redux/Store/Settings/Expansions/Expansions/index.ts

```diff
@@ -161,28 +146,68 @@ const getStandaloneExpansions = createSelector(
     getStandaloneExpansionIds,
     LanguageSelectors.getLanguagesByExpansion,
   ],
-  (content, ids, languages) =>
-    ids.map(id => getContentWithLanguageFallback(languages, content, id))
+  (content, ids, languages) => {
+    let expansions: types.Expansion[]
```

Comment:
These `let` bindings seem a bit unnecessary (as well as the type annotation)

Could you please change this to

```ts
    const expansions = ids.map((id) =>
      getContentWithLanguageFallback(languages, content, id)
    )
    return sortExpansions(expansions)
```